en/scripting/modules/index.md

```diff
@@ -1,88 +1,86 @@
-# Module
+# Modules
 
-Engine and editor expose their functionalities API through modules. Modules are in form of ECMAScript module format.
+Engines and editors expose their functionality through API interfaces and modules that exist in the form of **ECMAScript** modules.
 
-⚠️ Note, starting from 3.0, you can not access API through global variable `cc`!
+> **Note**: starting from 3.0, engine functions cannot be accessed through the global variable, prefixed with `cc`!
```

Comment:
......through the global variable prefixed with `cc`!

Reply:
what about:

starting from 3.0, engine functions cannot be accessed through global variables, prefixed with `cc`!

we would prefer a pause here if we are reading.